src/Service/Fido2ServiceEndpoints.cs

```diff
@@ -258,6 +252,27 @@ public async Task<TokenResponse> RegisterComplete(RegistrationCompleteDTO reques
         return new TokenResponse(token);
     }
 
+    public Task<string> CreateSigninToken(string userId)
+    {
+        ValidateUserId(userId);
```

Comment:
Not something that needs to be done now, but do we need to check to see if there's an existing credential? And then invalidate said credential?  That would be something to consider.